tests/visualization_tests/test_parallel_coordinate.py

```diff
@@ -31,21 +31,21 @@ def test_plot_parallel_coordinate() -> None:
     figure = plot_parallel_coordinate(study)
     assert len(figure.data[0]["dimensions"]) == 3
     assert figure.data[0]["dimensions"][0]["label"] == "Objective Value"
-    assert figure.data[0]["dimensions"][0]["range"] == (0.0, 2.0)
-    assert figure.data[0]["dimensions"][0]["values"] == (0.0, 2.0, 1.0)
+    assert figure.data[0]["dimensions"][0]["range"] == (0.0, 1.0)
```

Comment:
```
study.trials_dataframe()
   number  value             datetime_start          datetime_complete duration  params_param_a  params_param_b     state
0       0    0.0 2022-03-16 09:52:21.366738 2022-03-16 09:52:21.366738   0 days             1.0             2.0  COMPLETE
1       1    2.0 2022-03-16 09:52:21.366833 2022-03-16 09:52:21.366833   0 days             NaN             0.0  COMPLETE
2       2    1.0 2022-03-16 09:52:21.366892 2022-03-16 09:52:21.366892   0 days             2.5             1.0  COMPLETE
```

[note] `study` generated by testing utility has a trial with missing param, which arises a need to change existing tests.